docs/infrastructure_and_maintenance/cache/persistence_cache.md

```diff
@@ -210,79 +210,9 @@ Several cloud providers have managed services that are easier to set up, handle
-
-!!! note "[[= product_name_cloud =]]"
-
-    For [[= product_name_cloud =]]/Platform.sh: This is automatically configured in `vendor/ibexa/core/src/bundle/Core/DependencyInjection/IbexaCoreExtension.php` if you have enabled Memcached as `cache` Platform.sh service.
-
-For anything else, you can enable it with environment variables detected automatically by `vendor/ibexa/core/src/bundle/Core/DependencyInjection/IbexaCoreExtension.php`.
-For instance, if you set the following environment variables `export CACHE_POOL="cache.memcached" CACHE_DSN="user:pass@localhost?weight=33"`, it results in config like this:
-
-``` yaml
-services:
-    cache.memcached:
-        parent: cache.adapter.memcached
-        tags:
-            - name: cache.pool
-              clearer: cache.app_clearer
-              provider: 'memcached://user:pass@localhost?weight=33'
-              # Default CACHE_NAMESPACE value, see bin/config/cache_pool/cache.redis.yaml for usage with e.g. multi repo.
-              namespace: 'ez'
-```
-
-See `config/default_parameters.yaml` and `config/cache_pool/cache.memcached.yaml` for further details on `CACHE_POOL`, `CACHE_DSN` and `CACHE_NAMESPACE`.
-
-!!! caution "Clearing Memcached cache"
-
-    The regular `php bin/console cache:clear` command doesn't clear Memcached persistence cache.
-    Use a dedicated Symfony command to clear the pool you have configured: `php bin/console cache:pool:clear cache.memcached`.
-
-
-!!! caution "Connection errors issue"
-
-    If Memcached does display connection errors when using the default (ascii) protocol, then switching to binary protocol *(in the configuration and Memcached daemon)* should resolve the issue.
-
-!!! note
-
-    Memcached must not be bound to the local address if clusters are in use, or user logins fail.
-    To avoid this, in `/etc/memcached.conf` take a look under `# Specify which IP address to listen on. The default is to listen on all IP addresses`
-
-    For development environments, change the address below this comment in `/etc/memcached.conf` to `-l 0.0.0.0`
-
-    For production environments, follow this more secure instruction from the Memcached man:
-
-    > -l &lt;addr&gt;
-
-    > Listen on &lt;addr&gt;; default to INADDR\_ANY. &lt;addr&gt; may be specified as host:port.
-    If you don't specify a port number, the value you specified with -p or -U is used.
-    You may specify multiple addresses separated by comma or by using -l multiple times.
-    This is an important option to consider as there is no other way to secure the installation.
-    Binding to an internal or firewalled network interface is suggested.
-
 ## Using cache service
 
-Using the internal cache service allows you to use an interface and without caring whether the system is configured to place the cache in Memcached or on File system.
+Using the internal cache service allows you to use an interface and without caring whether the system is configured to place the cache on File system.
```

Comment:
```suggestion
Using the internal cache service allows you to use an interface and without caring whether the system is configured to place the cache in Redis or on File system.
```
I know this sentence could be probably rewritten, but I'd keep the enumaration anyway - it highlights the importance of using the cache service abstraction.